Решил

vzeten opened the pull request.

Commit:
Решил

iBerbasov commented on src/main/java/school/faang/bjs2_90472_Spotify/Main.java:
```diff
@@ -0,0 +1,19 @@
+        user3.start();
+
+        Thread user4 = new Thread(player::previous);
+        user4.start();
```
Имена с порядковыми цифрами это всегд очень плохой выбор для переменных
Лучше вынести все эти повторяющиеся действия в цикл и там запускать потоки

iBerbasov commented on src/main/java/school/faang/bjs2_90472_Spotify/Player.java:
```diff
@@ -0,0 +1,49 @@
+        lock.lock();
+        try {
+            System.out.println("Начал воспроизведение музыки");
+            isPlaying = true;
```
Перед изменением состояния надо проверять, стоит ли сейчас пауза или нет. И изменять состояние, когда это требуется

Commit:
Добавил проверку на то, прогирывается ли трек в данный момент, пытаюсь создать новый цикл

Commit:
Создал лист потоков
создал лист задач
запустил цикл по задачам создавая на каждую новый поток
приосединился к каждому потоку через цикл
словил ошибку прерывания и объявил об этом
восстановил флаг прерывания
